src/test/java/hudson/plugins/ansicolor/AnsiColorStepTest.java

```diff
@@ -228,6 +228,14 @@ public void canUseAndReportDefaultColorMapName() {
         );
     }
 
+    @Issue("JENKINS-66684")
+    @Test
+    public void canGetConstructorParametersForSnippetGenerator() {
+        final String colorMapName = AnsiColorMap.VGA.getName();
+        final AnsiColorStep step = new AnsiColorStep(colorMapName);
+        assertThat(step.getColorMapName()).isEqualTo(colorMapName);
```

Comment:
There are actually better test utilities in `workflow-step-api` and `workflow-cps`.